app/views/_controls.html.erb

```diff
@@ -0,0 +1,17 @@
+<div id="masthead_controls" class="container-fluid">
+  <div class="row">
+    <div class="col-xs-12 col-sm-5 col-md-6">
+      <nav class="navbar navbar-default" role="navigation">
```

Comment:
I am overriding the template found here which has main nav bar items: https://github.com/projecthydra/sufia/blob/master/app/views/_controls.html.erb and I'm replacing with the main navigation shown in the design. I'll have to see how to add About/Help/Contact as a secondary navigation...